map/bookmark_manager.cpp

```diff
@@ -1514,11 +1514,7 @@ bool BookmarkManager::IsSearchAllowed(kml::MarkGroupId groupId) const
       indexedBookmarksCount += it->second->GetUserMarks().size();
   }
 
-  /// @todo This function is actually called on Android only. Probably, there was some problems
-  /// with indexing large sets of bookmarks, but iOS works good at the same time?
-  auto const bookmarksCount = GetUserMarkIds(groupId).size();
-  auto const maxCount = SearchAPI::GetMaximumPossibleNumberOfBookmarksToIndex();
-  return indexedBookmarksCount + bookmarksCount <= maxCount;
+  return true;
```

Comment:
I wrote this comment :)
Actually, I did an investigation before, but decided to postpone changes (can't remember why).
If you feel the force to go deeper :) ->
- indexedBookmarksCount calculation is also not needed
- Should check calls for IsSearchAllowed and why we have IsIndexingOfBookmarkGroupEnabled

Reply:
Thanks for the heads up about that! I decided to dig deeper and removed the indexedBookmarksCount calculation. Being so closely related, I also removed IsSearchAllowed and IsIndexingOfBookmarkGroupEnabled methods where necessary. There should be no traces left now about this I am hopeful.